llama-index-integrations/vector_stores/llama-index-vector-stores-azureaisearch/pyproject.toml

```diff
@@ -32,7 +32,7 @@ version = "0.2.0"
 
 [tool.poetry.dependencies]
 python = ">=3.8.1,<4.0"
-azure-search-documents = "^11.4.0"
+azure-search-documents = "^11.5.1"
```

Comment:
lets also bump the version of the overall package to `0.2.1` maybe? (It will publish on merge then)